### PR TITLE
Adds hint to readme on downloading latest build

### PR DESCRIPTION
**Pull request by JosephSalisbury**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,8 @@
 # architect
 
-`architect` is a tool for managing Giant Swarm release engineering
\ No newline at end of file
+`architect` is a tool for managing Giant Swarm release engineering
```
Comment (xh3b4sd) [inline README.md]: Full stop. It is a fucking sentence.

File: README.md [hunk 2]
```diff
@@ -1,3 +1,8 @@
...
+
+To fetch the latest build of `architect`, run:
+```
+wget -q $(curl -sS https://api.github.com/repos/giantswarm/architect/releases/latest | grep browser_download_url | head -n 1 | cut -d '"' -f 4)
```
Comment (xh3b4sd) [inline README.md]: Where do we need to do this and who needs to do it? This has to be done in the circle configs in the projects who want to build via architect, right?

Comment (JosephSalisbury) [inline README.md]: Yeah, I'm putting it here for documentation. It will make up part of the unified circle.yml

File: README.md (hunk 1 above)
Comment (JosephSalisbury) [inline README.md]: Feel free to submit a PR :D

**Pull request merged**